README.md

```diff
@@ -1 +1,134 @@
-# hello_rails_back_end
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Rails React Starter <a name="about-project"></a>
+
+## [Hello React Front End]()
+
+It is a simple web application demonstrating use of React with Rails.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby-on-Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgres</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Fetch and dispay greetings from API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: any web-browser.
+
+### Setup
+
+Clone this project to your desired folder.
+
+```
+    git clone https://github.com/shafiuyushawu/hello_rails_back_end.git
+```
+
+### Install
+
+Run from a terminal
+
+```
+    cd rails_react_starter
+
+    bundle install
+```
+
+### Usage
+
+**To run the project follow the following instruction:-**
+
+```
+  rails db:drop
+
+  rails db:create
+
+  rails db:migrate
+
+  rails server
+```
```

Comment:
- [ ] [OPTIONAL] Kindly add 
``` ruby
rails db:seed
```
so the end user can seed the greetings you have in the DB

Reply:
@Whoistolu 
Thanks for the suggestion.